views/layout.hbs

```diff
@@ -1,13 +1,23 @@
-<!DOCTYPE html>
 <html>
   <head>
     <title>{{title}}</title>
-    <link rel='stylesheet' href='/stylesheets/style.css' />
+    <link rel="stylesheet" href="/stylesheets/style.css" />
   </head>
   <body>
-    <form action='/sessions?_method=DELETE' method='POST'>
-      <input type="submit" value="Log Out">
-    </form>
+    <p>
+      <a href="/posts"> Acebook </a>
```

Comment:
The PR looks good to me code-wise/logic, an optional suggestion: maybe remove the spaces around the links so the space after the word(s) is not clickable, e.g. `>Acebook</a>`